VERSION

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
I think it's easier to reason about having the file there, especially since it's needed for every build (including non-tagged builds). Getting the most recent tag and then updating it depending on whether it's a dev or release build seems kinda messy.

The plan is to add a 'tag-release' script which sets the appropriate version in that file, git tags and commits, and then bumps the minor version again for the current "dev" version.

There's room for improvement, but I don't see a problem with a VERSION file.